docs/SENTENCES.md

```diff
@@ -34,8 +42,26 @@ Feel free to set up this QA however makes most sense for you, but here's a [samp
 
 Once the review is complete, submit a pull request with the # of sentences submitted, a link to the manual QA results, and the % error rate. Here's [an example PR](https://github.com/mozilla/common-voice/pull/2873). Please make sure the sentences are in a plain `.txt` file with one sentence per line.
 
-## Flagging problematic sentences
+## QA that applies (or not) to the different imputs and outputs
```

Comment:
```suggestion
## QA that applies (or not) to the different inputs and outputs
```

however I'm not sure this title is fully understandable. Maybe "Cleanup/Processing" might be better? Not really convinced of that either though.